picard/tagger.py

```diff
@@ -1085,30 +1091,35 @@ def main(localedir=None, autoupdate=True):
 
     signal.signal(signal.SIGINT, signal.SIG_DFL)
 
-    picard_args, unparsed_args = process_picard_args()
+    picard_args = process_picard_args()
     if picard_args.version:
         return version()
     if picard_args.long_version:
         return longversion()
 
-    try:
-        from PyQt5.QtDBus import QDBusConnection
-        dbus = QDBusConnection.sessionBus()
-        dbus.registerService(PICARD_APP_ID)
-    except ImportError:
-        pass
-
-    tagger = Tagger(picard_args, unparsed_args, localedir, autoupdate)
+    PICARD_VER_NO_DOTS = PICARD_FANCY_VERSION_STR.replace(".", "-")
```

Comment:
I guess this conversion is needed to build a file path for Windows, so I think we should pass the version as is, and sanitize it at file path building time, the removal of dots is likely only needed on Win platforms.

Reply:
New commit should handle it correctly